### DISCUSSION TITLE
Koi Devnet Sunset

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1640] [General]
## Koi Devnet Sunset Announcement

The Koi Devnet will be sunset by the end of 2024, with the Crab chain transitioning to serve as the long-term incentivized testnet, supporting more advanced testing functionalities.

As Darwinia chain development stabilizes, this transition aims to reduce DevOps and operational overhead while simplifying the testing network landscape for developers. Additionally, the Koi Devnet has seen minimal usage recently, making its maintenance less justifiable. For runtime developers requiring more agile testing environments, local setups such as `localnet` or `devnet` can be used for simulation and testing. Furthermore, the Substrate ecosystem offers tools like Chopsticks, which facilitate local forking and provide enhanced testing flexibility.

---

### **Actions to Do**

- [x] **Twitter Announcement**: Notify developers about the sunset of Koi Devnet and encourage them to transition if they currently use Koi.  
- [x] **Faucet Close/Switch and Support for Crab Chain**: Close the Koi faucet and set up a guide for the Crab chain faucet if necessary. Ensure rate limits are implemented, or switch to a manual request process where users provide addresses via a forum.  
- [x] **Update Documentation**: Revise all relevant documentation to reflect these changes and guide developers in transitioning from Koi to Crab.  
- [x] **DevOps Cleanup**: Decommission and remove all infrastructure and resources related to Koi Devnet.  
- [x] **Remove Koi Blockscout**: Shut down the Koi block explorer at [koi-scan.darwinia.network](https://koi-scan.darwinia.network/).  
- [x] **Remove Koi runtime code from main branch**: https://github.com/darwinia-network/darwinia/tree/main/runtime/koi , history version can be found on releases, and archive one node db if necessary.

**Comment by boundless-forest** [comment]
Darwinia docs updated. https://github.com/darwinia-network/docs/pull/117
